docs/advanced-topics/networktime-ticks.md

```diff
@@ -0,0 +1,204 @@
+---
+id: networktime-ticks
+title: NetworkTime and Ticks
+sidebar_label: NetworkTime & Ticks
+---
+
+## LocalTime and ServerTime
+
+Why are there two different time values and which one should be used?
+
+Netcode for Gameobjects uses a star topology. That means all communications happen between the clients and the server/host and never between clients directly. Messages take time to transmit over the network. That's why RPCs and NetworkVariable will not happen immediately on other machines. NetworkTime allows to use time while considering those transmission delays.
```

Comment:
I wanted to link to this here but couldn't figure out how:
https://docs-multiplayer.unity3d.com/docs/reference/glossary/network-topologies